definitions/variable/technology/electricity-operation.yaml

```diff
@@ -112,15 +119,35 @@
     description: Not served demand
     unit: MWh
 - Injected Energy|Electricity|{Electricity Storage Type}:
-    description: Amount of energy charged to the storage during the period
+    description: Amount of energy charged from the grid to the storage during the period
     notes: this represents the total energy which is charged into the storage during the period, the energy which is discharged during 
      that same period is not substracted from this total. This means that the Net Charged Energy can be computed as Charged Energy - Discharged Energy
      if Charged Energy > Discharged Energy, 0 if not
     unit: GWh
 - Discharged Energy|Electricity|{Electricity Storage Type}:
-    description: Amount of energy charged to the storage during the period
-    notes: this represents the total energy which is discharged from the storage during the period, the energy which is charged into the storage during 
+    description: Amount of energy discharged from the storage to the grid during the period
+    notes: this represents the total energy which is discharged from the storage to the grid during the period, the energy which is charged into the storage during 
      that same period is not substracted from this total. This means that the Net Discharged Energy can be computed as Discharged Energy - Charged Energy,
      if Discharged Energy > Charged Energy, 0 if not
-
     unit: GWh
+- Injected Energy|Electricity|Load Shifting|{Residential Load Shifting Asset}:
+    description: Surplus of energy consumed by the {Residential Load Shifting Asset} during the period,
```

Comment:
Surplus, or is it just the amount of energy consumed?
I only try to homogenize with the term used in "Discharged Energy|..."

Reply:
yes, it is surplus. The principle is that a load shifting asset, such as a washing machine already has a nominal consumption profile. When activating the load shifting you will either increase or decrease its consumption in a certain period (and decrease resp. increase it in aanother period, so that the consumption remains the same on a given 'window' (eg 12h for washing machine, 2hours for air conditioning....). The load shifting asset is thus modelled as a virtual storage; from the point of view of this "storage" we could write amount of energy consumed but this may be very ambiguous. (you can have a look at CS1 where this modelling is described )

Reply:
I also agree with it. "Surplus" fits well since it is an increase or decrease from the consumption made or expected.